prov/sm2/src/sm2.h

```diff
@@ -218,7 +219,7 @@ struct sm2_domain {
 	struct fid_peer_srx *srx;
 };
 
-#define SM2_PREFIX "fi_shm://"
+#define SM2_PREFIX "fi_sm2://"
```

Comment:
Not related to nemesis I don't think. Move to a separate patch that addresses all lingering shms that got missed during the duplication

Reply:
1 patch